src/components/PopupWithForm.js

```diff
@@ -25,14 +25,16 @@ export default class PopupWithForm extends Popup {
 
     this._popupForm.addEventListener("submit", (event) => {
       event.preventDefault();
-      this._handleFormSubmit(this._getInputValues());
+      const inputValues = this._getInputValues();
+      console.log("Input Values:", inputValues);
+      this._handleFormSubmit(inputValues);
 
-      this.close();
+      // this.close();
     });
   }
 
-  // close() {
-  //   this._popupForm.reset();
-  //   super.close();
-  // }
+  close() {
+    // this._popupForm.reset();
+    super.close();
```

Comment:
You shouldn’t create the **same methods** of the parent class if you don’t change them inside and only call them with `super`. The parent methods are available in any inheritor of the class.